zorg/jenkins/jobs/jobs/lldb-statistics

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env groovy
+
+// These are the LLVM build directories involved:
+//
+// (a) The "host" compiler/LLDB is taken from whatever the LLDB incremental built/used.
+//     The metrics we collect are from that "host" LLDB that we fetched. Throughout this
+//     file it is commonly referred to as 'host-compiler' or 'HOST_BUILD_DIR'.
+// 
+// (b) The debugger/compiler that we're debugging as part of metrics collection is pinned to
+//     the llvm-19.x release and is supposed to be unchanging across runs of this job.
+//     Throughout this file we refer to it as "historic" or 'clang-19'.
+// 
+// (c) We use the compiler from (a) to build the "historic" Clang/LLDB.
+// 
+// (d) The compiler compiling the debugger in (a) is the Clang produced by the clang-stage2
+//     buildbot and happens outside the purview job.
+//
+// In summary, the only stable version of Clang/LLVM is the one we use as the inferior during
+// metrics collection.
+
+pipeline {
+    options {
+        disableConcurrentBuilds()
+
+        timeout(time: 12, unit: 'HOURS')
+    }
+
+    parameters {
+        string(name: 'LABEL', defaultValue: params.LABEL ?: 'macos-x86_64', description: 'Node label to run on')
+        string(name: 'GIT_SHA', defaultValue: params.GIT_REVISION ?: '*/release/19.x', description: 'Git commit to build.')
+        string(name: 'ARTIFACT', defaultValue: params.ARTIFACT ?: 'lldb-cmake-intel/latest', description: 'Clang/LLDB artifact to use')
```

Comment:
I don't think this artifact exists right now

Reply:
Ah that was my worry. What would be the process of making it available? The only reference to publishing the artefacts that I could find is:
```
echo "ARTIFACT=$JOB_NAME/clang-d$GIT_DISTANCE-g$GIT_SHA-t$BUILD_ID-b$BUILD_NUMBER.tar.gz" >> build.properties```
in `jobs/clang-stage1-RA`. Is it some combination of a Jenkins configuration and this `build.properties`?